packages/teleport/src/Databases/AddDatabase/AddDatabase.test.tsx

```diff
@@ -16,23 +16,32 @@
-    ${'self-hosted mongodb'}  | ${'teleport db start --token=[generated-join-token] --auth-server=localhost:443 --name=[db-name] --protocol=mongodb --uri=[uri]'}
+    input                     | atemptStatus | output
+    ${'self-hosted mysql'}    | ${'failed'}  | ${'teleport db start --token=[generated-join-token] --auth-server=localhost:443 --name=[db-name] --protocol=mysql --uri=[uri]'}
+    ${'self-hosted mysql'}    | ${'success'} | ${'teleport db start --token=some-token --auth-server=localhost:443 --name=[db-name] --protocol=mysql --uri=[uri]'}
+    ${'rds mysql'}            | ${'failed'}  | ${'teleport db start --token=[generated-join-token] --auth-server=localhost:443 --name=[db-name] --protocol=mysql --uri=[uri] --aws-region=[region]'}
+    ${'rds mysql'}            | ${'success'} | ${'teleport db start --token=some-token --auth-server=localhost:443 --name=[db-name] --protocol=mysql --uri=[uri] --aws-region=[region]'}
+    ${'cloud sql mysql'}      | ${'failed'}  | ${'teleport db start --token=[generated-join-token] --auth-server=localhost:443 --name=[db-name] --protocol=mysql --uri=[uri] --ca-cert=[instance-ca-filepath] --gcp-project-id=[project-id] --gcp-instance-id=[instance-id]'}
+    ${'cloud sql mysql'}      | ${'success'} | ${'teleport db start --token=some-token --auth-server=localhost:443 --name=[db-name] --protocol=mysql --uri=[uri] --ca-cert=[instance-ca-filepath] --gcp-project-id=[project-id] --gcp-instance-id=[instance-id]'}
+    ${'self-hosted postgres'} | ${'failed'}  | ${'teleport db start --token=[generated-join-token] --auth-server=localhost:443 --name=[db-name] --protocol=postgres --uri=[uri]'}
+    ${'self-hosted postgres'} | ${'success'} | ${'teleport db start --token=some-token --auth-server=localhost:443 --name=[db-name] --protocol=postgres --uri=[uri]'}
+    ${'rds postgres'}         | ${'failed'}  | ${'teleport db start --token=[generated-join-token] --auth-server=localhost:443 --name=[db-name] --protocol=postgres --uri=[uri] --aws-region=[region]'}
+    ${'rds postgres'}         | ${'success'} | ${'teleport db start --token=some-token --auth-server=localhost:443 --name=[db-name] --protocol=postgres --uri=[uri] --aws-region=[region]'}
+    ${'cloud sql postgres'}   | ${'failed'}  | ${'teleport db start --token=[generated-join-token] --auth-server=localhost:443 --name=[db-name] --protocol=postgres --uri=[uri] --ca-cert=[instance-ca-filepath] --gcp-project-id=[project-id] --gcp-instance-id=[instance-id]'}
+    ${'cloud sql postgres'}   | ${'success'} | ${'teleport db start --token=some-token --auth-server=localhost:443 --name=[db-name] --protocol=postgres --uri=[uri] --ca-cert=[instance-ca-filepath] --gcp-project-id=[project-id] --gcp-instance-id=[instance-id]'}
+    ${'redshift'}             | ${'failed'}  | ${'teleport db start --token=[generated-join-token] --auth-server=localhost:443 --name=[db-name] --protocol=postgres --uri=[uri] --aws-region=[region] --aws-redshift-cluster-id=[cluster-id]'}
```

Comment:
lol i think this is overkill, especially since we have a snapshot somewhere where we test for correct rendering. i think how the test was before was fine. The only thing that changes is the token value (all the success tests pretty much tested that our generateCmd formats things as expected) :shrug: